src/lib/containers/QueryWrapperMenu.tsx

```diff
@@ -99,9 +122,19 @@ export default class QueryWrapperMenu extends React.Component<QueryWrapperMenuPr
   public render() {
     const menuDropdown = this.renderFacetMenu()
     const queryWrapper = this.renderQueryChildren()
-
+    const name = window.location.hash.substring('#/Explore/'.length) || 'Grants'
```

Comment:
substring of 10??? ('#/Explore/'.length)

Reply:
I can change that, I was thinking 'that would be too hardcoded' but now that I think about it, thats no more hard coded than the actual string itself

Reply:
whoops, I definitely didn't mean for grants to be the default value